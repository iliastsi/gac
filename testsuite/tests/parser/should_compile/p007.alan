hello() : proc
   c : int;	
{
  (* this is a block *)
  {
	c = 40+2;
	c = 6*7;;
 { c = 5;}
  {}}
}
