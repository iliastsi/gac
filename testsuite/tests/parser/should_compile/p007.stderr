==================== Parser ====================
hello() : proc
  c : int;
{ -- hello
  {
    c = 40 + 2;
    c = 6 * 7;
    ;
    {
      c = 5;
    }
    {
    }
  }
} -- hello


